### PR TITLE
Add individual testing for testing from System.in

### PR DESCRIPTION
**Pull request by goodudetheboy**
I added an extra form of testing for OpeningHoursParser. Individual testing receives input directly from `System.in` for on-the-fly parsing. To use this test, run `gradle individualTesting` for non-strict mode or `gradle individualTestingStrict` for strict mode. 

For best aesthetic, use `gradle individualTesting--console=plain` for non-strict mode or `gradle individualTestingStrict --console=plain` for strict mode. It removes the progress bar from gradle, and looks better I guess.

### CODE REVIEW

**Comment by goodudetheboy** [comment]
I think this test might be useful if we just want to have a fast look at the parse result from an input string, directly from the terminal.

File: build.gradle [hunk 1]
```diff
@@ -106,6 +107,21 @@ task showDifferencesStrict(type: JavaExec) {
     classpath = sourceSets.test.runtimeClasspath
     args('test-data/oh.txt-result-strict', 'test-data/oh.txt-result-strict-temp', 'test-data/oh.txt', 'test-data/diff-strict.txt')
 }
+
+task individualTesting(type: JavaExec) {
```
Comment (simonpoole) [inline build.gradle:111]: Both new tasks should be in the verification group or else they are rather hard to find in UIs that display tasks.

File: build.gradle [hunk 2]
```diff
@@ -106,6 +107,21 @@ task showDifferencesStrict(type: JavaExec) {
...
+    args('false')
+}
+
+task individualTestingStrict(type: JavaExec) {
+    main = "ch.poole.openinghoursparser.IndividualTest"
+    classpath = sourceSets.test.runtimeClasspath
+    standardInput = System.in
+    args('true')
+}
+
```
Comment (simonpoole) [inline build.gradle:124]: Both tasks should be moved below line 126 as 125 and 128 belong to the showDifferencesStrict task

File: src/test/java/ch/poole/openinghoursparser/IndividualTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    
+            } catch (OpeningHoursParseException e) {
+                System.out.println("Illegal input string");
+                e.printStackTrace();
+                System.out.println();
+                System.out.println("------------------------------");
+                System.out.println();
+            }
+        }
+
```
Comment (simonpoole) [inline src/test/java/ch/poole/openinghoursparser/IndividualTest.java:49]: Lots of gratuitous empty lines here, for what every reason I find them visually extremely annoying between braces, could you remove them, thanks.

**Review by simonpoole — CHANGES_REQUESTED** [review]

**Review by goodudetheboy — COMMENTED** [review]
Changes added according to your reviews.

**Comment by simonpoole** [comment]
> 
> 
> Changes added according to your reviews.

Could you rebase on current master (there seems to be some weird commit history that needs cleaning up) and I'll merge.

**Review by simonpoole — APPROVED** [review]

**Comment by goodudetheboy** [comment]
I have rebased in 51f38a02892aba9e5ee9407e463459dc79955851 and cleaned up some files in a921da4f0fc8073feb813b8ff539873512749f0c, so my current head should be even with master.

**Comment by simonpoole** [comment]
Thanks!

**Pull request merged**